app/mailers/patient_mailer.rb

```diff
@@ -15,6 +15,8 @@ def enrollment_email(patient)
     end
     @lang = patient.select_language(:email)
     @contact_info = patient.jurisdiction.contact_info
+    @custom_enrollment_message = patient&.jurisdiction&.custom_messages_for_hierarchy&.dig('assessments', 'html', 'email', 'info1', @lang&.to_s)
```

Comment:
in case there are any future custom messages for emails, this would be more specific and also follows the pattern in the locale files

```suggestion
    @custom_enrollment_message = patient&.jurisdiction&.custom_messages_for_hierarchy&.dig('assessments', 'html', 'email', 'enrollment', 'info1', @lang&.to_s)
```

Reply:
Good catch, meant to follow the structure of the locale file so this was just a mistake.

Reply:
fixed: https://github.com/SaraAlert/SaraAlert/pull/1159/commits/23c5860dbe788d5b5f260e3919c819a8c5d21580